### PR TITLE
Add new welcome messages

### PR DESCRIPTION
**Pull request by SergioJThompson**
Add four new welcome messages

### CODE REVIEW

**Comment by SergioJThompson** [comment]
I'm new to Github and this is my first attempt at contributing to someone else's repository, forgive me for any mistakes.

File: crawl-ref/source/dat/database/miscname.txt [hunk 1]
```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
 A smirking vizier told you that the Orb of Zot is just within this dungeon.
 
 If you return without finding the Orb of Zot, so far below, it's all over!
+
+One, two, three... Ready or not, find the Orb of Zot!
```
Comment (PleasingFungus) [inline crawl-ref/source/dat/database/miscname.txt:140]: I get what you're going for here, but this is a bit out of tone for the game. Crawl can be silly, but it's not usually playground-rhymes silly. Every welcome message could be the very first thing a new player sees, and this isn't a great introduction for them.

File: crawl-ref/source/dat/database/miscname.txt [hunk 2]
```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
...
+
+Far below lies the Orb of Zot. Hopefully nothing unpleasant happens on the way there.
+
+It's time to claim the Orb of Zot, or maybe just get slain and rot!
```
Comment (PleasingFungus) [inline crawl-ref/source/dat/database/miscname.txt:144]: This is pretty similar to an existing line:

> Are you destined to find the @_great_adj_@ Orb of Zot, or to die trying?

Which is fine! But this version is a bit more over-the-top (get slain AND rot!) - it's excessive.

File: crawl-ref/source/dat/database/miscname.txt [hunk 3]
```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
...
+
+The @_great_adj_@ Orb of Zot should be around here somewhere.
```
Comment (PleasingFungus) [inline crawl-ref/source/dat/database/miscname.txt:146]: This is just wrong, unfortunately. Maybe that's the joke, but it's not 'around here somewhere' - it's quite a long way below. Check out the other messages - it's "in the heart of the dungeon", "in the unfathomable depths", "deep within this dungeon". The only one which is a little more misleading is "A smirking vizier told you that the Orb of Zot is just within this dungeon", and that one pretty explicitly presents itself as being from an untrustworthy source.

As the first message we give a new player, let's not lie to them without any warning.

File: crawl-ref/source/dat/database/miscname.txt [hunk 4]
```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
 A smirking vizier told you that the Orb of Zot is just within this dungeon.
 
 If you return without finding the Orb of Zot, so far below, it's all over!
+
+One, two, three... Ready or not, find the Orb of Zot!
+
+Far below lies the Orb of Zot. Hopefully nothing unpleasant happens on the way there.
```
Comment (PleasingFungus) [inline crawl-ref/source/dat/database/miscname.txt:142]: I like the concept of this one! The tone is a little bit too casual, though, and the opener is almost exactly the same as one above ("Below lies the Orb of Zot..."). I've tweaked it a bit.

**Review by PleasingFungus — COMMENTED** [review]
Hi, thanks for the submission! No need to apologize about anything - you did the submission just fine.

I've merged one of your welcome messages in https://github.com/crawl/crawl/commit/694778e40971512d97c6d68af53640ade87a86f9 . I left some thoughts about the others - hopefully that's helpful! :)

**Pull request closed without merging**